python/pyspark/sql/connect/session.py

```diff
@@ -1072,6 +1083,13 @@ def _start_connect_server(master: str, opts: Dict[str, Any]) -> None:
                 conf.setAll(list(overwrite_conf.items())).setAll(list(default_conf.items()))
                 PySparkSession(SparkContext.getOrCreate(conf))
 
+                # In Python local mode, session.stop does not terminate JVM itself
```

Comment:
How about you set the environment variable when we start spark?

Reply:
Also how do you ensure the LocalAuthInterceptor will installed if there is no token yet?

Reply:
> How about you set the environment variable when we start spark?

Yeah I think that's a good idea.

> Also how do you ensure the LocalAuthInterceptor will installed if there is no token yet?

It is ensured by being blocked SparkConext with Spark Connect server is running. And then, here we create a Spark Connect client. So the token will always be there.